plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/CachingHiveMetastore.java

```diff
@@ -324,6 +325,17 @@ private static <K, V> V get(LoadingCache<K, V> cache, K key)
         }
     }
 
+    private static <K, V> V get(LoadingCache<K, V> cache, K key, Callable<? extends V> loader)
```

Comment:
Since we provide a loader, there is no point in requiring the cache to be `LoadingCache`. Drop "Loading".

BTW is this method equivalent to `io.trino.collect.cache.CacheUtils#uncheckedCacheGet`?

Reply:
switched to `uncheckedCacheGet`.

`uncheckedCacheGet` should probably use `e.getCause()` as `getUnchecked` does